cogs/welcome/welcome.py

```diff
@@ -318,6 +339,37 @@ async def logServerLeave(self, leaveUser: discord.Member):
                     leaveUser.id,
                 )
 
+    async def generateRandWelcomeImg(self, user):
+        """creates an image for the specific player using their avatar and an image from the random image pool, then returns it"""
+        base = Image.open(os.path.join(self.img_dir, random.choice(os.listdir(self.img_dir))))
+        mask = Image.open(os.path.join(os.path.dirname(os.path.realpath(__file__)), "MASK.png"))
```

Comment:
Here and everywhere you use the PNGs you've bundled, can you please put this in a `data/` subdirectory and use `data_manager.bundled_data_path(self)` to retrieve the path?